cmake/Modules/winconf-msvc.cmake

```diff
@@ -1,21 +1,12 @@
 if (STACKTRACE)
-    if ("${WINXXBITS}" STREQUAL "Win64")
-        add_compile_options(-Zi)
-    else ("${WINXXBITS}" STREQUAL "Win64")
+    if (NOT "${WINXXBITS}" STREQUAL "Win64")
         # i686 arch requires frame pointer preservation
         add_compile_options(-Oy-)
-    endif ("${WINXXBITS}" STREQUAL "Win64")
-    link_libraries(dbghelp.lib)
+    endif (NOT "${WINXXBITS}" STREQUAL "Win64")
+    add_compile_options(-Zi)
+    link_libraries(dbghelp.lib /DEBUG)
```

Comment:
cmake is taking `/DEBUG` as a library. Cmake MSVC builds are failing because of this.

Reply:
Oops... CMake isn't so intuitive...
I have no MSVC environment currently. Can you try it with `-DEBUG` instead?

Reply:
yeap it works....